library/consul

```diff
@@ -6,10 +6,10 @@ GitRepo: https://github.com/hashicorp/docker-consul.git
 GitCommit: 9cb67358af03c73222da3badebc7c69d765c456a
 Directory: 0.X
 
-Tags: 1.6.4, 1.6
+Tags: 1.6.5, 1.6
 Architectures: amd64, arm32v6, arm64v8, i386
 GitRepo: https://github.com/hashicorp/docker-consul.git
-GitCommit: 59bdf0ae8863e3166aadc81e18f8781a52a07136
+GitFetch: refs/heads/consul-1.6.5
```

Comment:
This still needs a `GitCommit` along with the `GitFetch` so that it knows which commit on that branch.

Reply:
Thank you for your help!